test/integration/run/test_run.py

```diff
@@ -36,25 +32,65 @@ def test_running_flag_version(tt_cmd, tmpdir):
         stdout=subprocess.PIPE,
         text=True
     )
-    start_output = instance_process.stdout.readline()
-    assert re.search(r"   • Running an instance...   \n", start_output)
     run_output = instance_process.stdout.readline()
     assert re.search(r"Tarantool", run_output)
 
 
 def test_running_flag_eval(tt_cmd, tmpdir):
+    # Start an instance.
+    start_cmd = [tt_cmd, "run", "-e", "print('123')"]
+    instance_process = subprocess.Popen(
+        start_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    run_output = instance_process.stdout.readline()
+    assert re.search(r"123", run_output)
+
+
+def test_running_arg(tt_cmd, tmpdir):
     # Copy the test application to the "run" directory.
+    test_app_path = os.path.join(os.path.dirname(__file__), "test_app", "test_app_arg.lua")
+    shutil.copy(test_app_path, tmpdir)
 
     # Start an instance.
```

Comment:
This comment is outdated. We are not starting instances with `run` .

Reply:
Changed